# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/PWebrtcProxyChannel.ipdl
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/ipc/PWebrtcProxyChannel.ipdl
--- a/media/mtransport/ipc/PWebrtcProxyChannel.ipdl	Tue Sep 17 18:15:23 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,38 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set sw=2 ts=8 et tw=80 ft=cpp : */
-
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-include protocol PNecko;
-include protocol PSocketProcess;
-
-include NeckoChannelParams;
-using mozilla::dom::TabId from "mozilla/dom/ipc/IdType.h";
-
-namespace mozilla {
-namespace net {
-
-async protocol PWebrtcProxyChannel
-{
-  manager PNecko or PSocketProcess;
-
-parent:
-  async AsyncOpen(nsCString aHost,
-                  int32_t aPort,
-                  LoadInfoArgs aLoadInfoArgs,
-                  nsCString aAlpn);
-  async Write(uint8_t[] aWriteData);
-  async Close();
-
-child:
-  async OnClose(nsresult aReason);
-  async OnConnected();
-  async OnRead(uint8_t[] aReadData);
-
-  async __delete__();
-};
-
-} // namespace net
-} // namespace mozilla